spipe/Makefile.BSD

```diff
@@ -78,8 +78,6 @@ SRCS	+=	crypto_entropy.c
 SRCS	+=	crypto_verify_bytes.c
 IDIRS	+=	-I../libcperciva/crypto
 
-crypto_aes_aesni.o: crypto_aes_aesni.c cpusupport-config.h
```

Comment:
I must confess that I'm not 100% certain that removing this is safe -- it appears that this code wasn't used at all?  Was this perhaps only used when doing `make -F Makefile.BSD all`?

Removing it has no change on the makefiles produced via `make Makefiles`, and spiped still compiles, and (on my system) the `-maes` option is is correctly passed to `crypto_aes_aesni.o` (and only that file).